src/Importer/DBImport.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace TheRealMVP\Importer;
+
+class DBImport
+{
+    private \PDO $pdoConnection;
+    private array $apiData;
```

Comment:
Suggest making these properties protected to allow possible extension in the future. (Open-Closed principle)